ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -20,7 +20,12 @@ const PredictionValue = styled.span`
   text-transform: uppercase;
 `
 
-const Prediction = ({ field, annotation, dangerThreshold, warningThreshold }) => {
+const LinkButton = styled.span.attrs({ role: 'button', tabIndex: '0' })`
```

Comment:
we have this as a utility already in `shared/components/buttons/ButtonLink`

Reply:
nice, I'll use that.